mmcv/cnn/__init__.py

```diff
@@ -10,14 +10,15 @@
     build_conv_layer, build_norm_layer, build_padding_layer,
     build_plugin_layer, build_upsample_layer, conv_ws_2d, is_norm)
 from .builder import MODELS, build_model_from_cfg
-# yapf: enable
 from .resnet import ResNet, make_res_layer
-from .utils import (INITIALIZERS, Caffe2XavierInit, ConstantInit, KaimingInit,
-                    NormalInit, PretrainedInit, TruncNormalInit, UniformInit,
-                    XavierInit, bias_init_with_prob, caffe2_xavier_init,
-                    constant_init, fuse_conv_bn, get_model_complexity_info,
-                    initialize, kaiming_init, normal_init, trunc_normal_init,
-                    uniform_init, xavier_init)
+from .utils import (
+    INITIALIZERS, Caffe2XavierInit, ConstantInit, KaimingInit,
+    NormalInit, PretrainedInit, TruncNormalInit, UniformInit,
+    XavierInit, bias_init_with_prob, caffe2_xavier_init,
+    constant_init, fuse_conv_bn, get_model_complexity_info,
+    initialize, kaiming_init, normal_init, trunc_normal_init,
+    uniform_init, xavier_init)
+# yapf: enable
```

Comment:
```suggestion
from .utils import (INITIALIZERS, Caffe2XavierInit, ConstantInit, KaimingInit,
                    NormalInit, PretrainedInit, TruncNormalInit, UniformInit,
                    XavierInit, bias_init_with_prob, caffe2_xavier_init,
                    constant_init, fuse_conv_bn, get_model_complexity_info,
                    initialize, kaiming_init, normal_init, trunc_normal_init,
                    uniform_init, xavier_init)
```

Reply:
Sorry, I force-pushed before seeing the recommended modification. It seems that the current version passes the lint check.

Reply:
Great.